src/server.ts

```diff
@@ -380,12 +380,17 @@ const createParticipant = async (tag: string): Promise<ParticipantInfo> => {
 const deleteSession = async () => {
   if (sessionId) {
     try {
-      await webRTCController.deleteSession(accountId, sessionId);
```

Comment:
Dumb question, but why did you remove this line? It looks to me that the only code in the try block is resetting the session Id and a console.log()

Reply:
that was a fat-finger mistake.

I sort of wondered where it went when I got to thinking "when does console.log throw an exception?"

It's back there now.